test/bits/pointer_memops_fail.c

```diff
@@ -0,0 +1,18 @@
+#include "smack.h"
+#include <stdlib.h>
+
+// @expect error
+// @flag --clang-options=-m32 --bit-precise-pointers
```

Comment:
Why do you need `-m32`? Does it time out otherwise?

Reply:
No, this is because we have a switch between 32 bit and 64 bit points and I want to test the 32 bit version.